R/citation_search_xDD.R

```diff
@@ -0,0 +1,55 @@
+#' Search for citations in xDD
+#'
+#' This function searches for citations in xDD. Uses 'snippets/term' function in xDD API and 
+#' searches through all of xDD corpus (not limited to full-text documents).
+#'
+#' @param identifiers a vector of identifiers to be searched for, without hypertext
+#' transfer protocol: "https://" or "http://" 
+#'
+#' @return tibble of publications and their identifiers that contain 
+#' @examples
+#' \dontrun{
+#' identifiers <- c("10.18739/A22274", "10.18739/A2D08X", "10.5063/F1T151VR")
+#' result <- citation_search_xDD(identifiers)
+#' }
+#'
+
+citation_search_xDD <- function(identifiers) {
+  
+  identifiers <- check_identifiers(identifiers)
+  
+  identifiers_enc <- lapply(identifiers, utils::URLencode, reserved = TRUE) # URLencode - percent-encode in URL
+  identifiers_enc <- unlist(identifiers_enc) # given a list, produces a vector
+  results <- list()
+  for (i in 1:length(identifiers_enc)) {
+    Sys.sleep(0.12)
```

Comment:
do you need to slow down the calls due to rate limiting or is this left over from another function that does need to?

Reply:
left over copied function, delete line 25